agent/tests/test_chat.py

```
@@ -38,52 +38,42 @@
 
 class TestChatCommand:
     @patch("iconfucius.cli.chat.run_chat")
-    def test_explicit_chat_command(self, mock_run_chat):
+    @patch("iconfucius.skills.executor.execute_tool", return_value={
+        "status": "ok", "config_exists": True, "wallet_exists": True,
+        "env_exists": True, "has_api_key": True, "ready": True,
+    })
+    def test_explicit_chat_command(self, mock_execute, mock_run_chat):
```

Ruff (0.15.4)

[warning] Unused method argument: `mock_execute`

(ARG002)